packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -13,20 +13,22 @@ type ProposalAction = (
 ) => Promise<MsgExecuteContract>
 
 export const wrapCommand = (command) => {
+  const address = process.env.CW3_FLEX_MULTISIG as AccAddress
```

Comment:
This will be undefined. The wrap function is executed before the environment is loaded

Reply:
Good catch!  Fixed